cineast-runtime/src/main/java/org/vitrivr/cineast/standalone/cli/TextRetrievalCommand.java

```diff
@@ -35,7 +35,6 @@ public void run() {
     retrievers.add(new OCRSearch());
     retrievers.add(new AudioTranscriptionSearch());
     retrievers.add(new DescriptionTextSearch());
-    retrievers.add(new TagsFtSearch());
```

Comment:
doesn't this one exist anymore?

Reply:
It's not part of the standard functionality a user can reasonably expect from any Cineast Instance since it searches a separate entity with a full-text index that has to be externally created.

Reply:
fair enough.